lib/stencil-init.js

```diff
@@ -115,19 +138,31 @@ class StencilInit {
                         return true;
                     }
                 },
-            },
-        ]);
+            });
+        }
+
+        return prompts.length > 0 ? prompts : null;
+    }
+
+    /**
+     * @param {{port: (number), normalStoreUrl: (string), accessToken: (string)}} defaultAnswers
+     * @param {{port: (number), url: (string), token: (string)}} cliOptions
+     * @returns {Promise<object>}
+     */
+    async askQuestions (questions) {
+        return await this.inquirer.prompt(questions);
```

Comment:
As for me, now this method looks a bit redundant so we can use `this.inquirer.prompt` directly.